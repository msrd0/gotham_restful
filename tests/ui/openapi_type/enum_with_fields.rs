#[macro_use]
extern crate gotham_restful;

#[derive(OpenapiType)]
enum Food {
	Pasta,
	Pizza { pineapple: bool },
	Rice,
	Other(String)
}

fn main() {}
