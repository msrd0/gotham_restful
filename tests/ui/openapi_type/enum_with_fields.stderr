error: #[derive(OpenapiType)] does not support enum variants with fields
 --> $DIR/enum_with_fields.rs:7:2
  |
7 |     Pizza { pineapple: bool },
  |     ^^^^^

error: #[derive(OpenapiType)] does not support enum variants with fields
 --> $DIR/enum_with_fields.rs:9:2
  |
9 |     Other(String)
  |     ^^^^^
